q2cli/tests/data/mapping_config.toml

```diff
@@ -4,15 +4,12 @@ strategy = "None"
 [[parsl.executors]]
 class = "ThreadPoolExecutor"
 label = "default"
-max_threads = 6
+max_threads = 1
```

Comment:
Does this help to resolve the socket leaks?

Reply:
No, but I had hoped it would when I was using HighThroughputExecutors.. ThreadPoolExecutors don't open ports, but HighThroughputExecutors do, the solution to the problem ended up being not using HighThroughputExecutors at all. This decrease is still useful though because there was no good reason to use a bunch of threads. It was just a waste of resources.